ci/builder/requirements.txt

```diff
@@ -36,6 +36,7 @@ pdoc==15.0.3
 # We can revert back to standard pg8000 versions once https://github.com/tlocke/pg8000/pull/161 is released
 pg8000@git+https://github.com/tlocke/pg8000@46c00021ade1d19466b07ed30392386c5f0a6b8e
 prettytable==3.16.0
+protobuf==5.29.3
```

Comment:
Specifically used this version since that's the version used in profile.proto

Reply:
This will be auto-upgraded by dependabot. Is that a problem?